libs/designer/src/lib/ui/templates/templateview.tsx

```diff
@@ -0,0 +1,100 @@
+import { useDispatch, useSelector } from 'react-redux';
+import type { CreateWorkflowHandler } from './TemplatesDesigner';
+import type { AppDispatch, RootState } from '../../core/state/templates/store';
+import { isMultiWorkflowTemplate, loadTemplate } from '../../core/actions/bjsworkflow/templates';
+import { useEffect } from 'react';
+import { TemplateOverview } from './templateoverview';
+import { setLayerHostSelector, Spinner, SpinnerSize, Text } from '@fluentui/react';
+import { CreateWorkflowPanel } from '../panel/templatePanel/createWorkflowPanel/createWorkflowPanel';
+import { QuickViewPanel } from '../panel/templatePanel/quickViewPanel/quickViewPanel';
+import { openCreateWorkflowPanelView, openQuickViewPanelView } from '../../core/state/templates/panelSlice';
+import { useIntl } from 'react-intl';
+
+export interface TemplateViewProps {
+  createWorkflow: CreateWorkflowHandler;
+  showSummary?: boolean;
+  showCloseButton?: boolean;
```

Comment:
Do we need these parameters? I thought for view template scenario, these values are fixed, so could we not expose them?

Reply:
I see, sounds good.